### ISSUE TITLE
Expected loss on S3DIS dataset

### ISSUE THREAD

**Issue by hhuang81** [LiuShihHung/GICN#11]
Hi, do you have the record of how your final loss looks on the S3DIS dataset? For example, for the pre-trained model you gave after 75 epochs, what's the magnitude of those different losses?

**Comment by kikuboy** [comment]
@hhuang81 Why do I train so slowly in the backward？

**Comment by meidachen** [comment]
I would really appreciate it if someone could provide their final losses.
